README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><a href="https://vuejs.org" target="_blank"><img width="100"src="https://vuejs.org/images/logo.png"></a></p>
+<p align="center"><a href="https://vuejs.org" target="_blank"><img width="100" src="https://vuejs.org/images/logo.png" alt="LOGO"></a></p>
```

Comment:
IMO _Vue logo_ would be a more precise description

Reply:
LOGO => IMO Vue logo ?